src/main/java/ch/wisv/areafiftylan/controller/TicketRestController.java

```diff
@@ -42,4 +52,22 @@ public TicketTransferRestController(TicketService ticketService) {
 
         return createResponseEntity(HttpStatus.OK, "Ticket transfer successfully cancelled");
     }
+
+    /**
+     * This method returns an overview of available tickets with some additional information
+     *
+     * @return A collection of all TicketTypes and their availability
+     */
+    @RequestMapping(value = "/tickets/available", method = RequestMethod.GET)
+    public Collection<TicketInformationResponse> getAvailableTickets() {
+        return orderService.getAvailableTickets();
+    }
+
+    @PreAuthorize("isAuthenticated()")
+    @RequestMapping(value = "/tickets/incontrol", method = RequestMethod.GET)
```

Comment:
I'm not sure I like the "incontrol" part. You're not really in control over the tickets, just the seat placement. Let's come up with something better